third-party-podspecs/react-native-blur.podspec.json

```diff
@@ -1,21 +1,23 @@
 {
-    "name": "react-native-blur",
-    "version": "3.3.0",
-    "summary": "Component implementation for UIVisualEffectView's blur and vibrancy effect.",
-    "description": "Component implementation for UIVisualEffectView's blur and vibrancy effect.",
-    "license": "MIT",
-    "authors": "react-native-community",
-    "homepage": "https://github.com/react-native-community/react-native-blur",
-    "source": {
-        "git": "https://github.com/react-native-community/react-native-blur",
-        "tag": "v3.3.0"
-    },
-    "source_files": "ios/*.{h,m}",
-    "preserve_paths": "**/*.js",
-    "platforms": {
-        "ios": "8.0"
-    },
-    "dependencies": {
-        "React": []
-    }
-}
\ No newline at end of file
+  "name": "react-native-blur",
+  "version": "0.8.0",
+  "source_files": "ios/*.{h,m}",
+  "platforms": {
+    "ios": "8.0",
+    "tvos": "9.0"
+  },
+  "authors": {
+    "Alexey Kureev": "kureev-mail@ya.ru"
+  },
+  "license": "MIT",
+  "summary": "Component implementation for UIVisualEffectView's blur and vibrancy effect.",
+  "homepage": "https://github.com/react-native-community/react-native-blur",
+  "source": {
+    "git": "https://github.com/react-native-community/react-native-blur.git"
```

Comment:
Also no tag or commit is defined in the latest version: https://github.com/Kureev/react-native-blur/blob/v3.6.1/react-native-blur.podspec#L15, which means this would depend on `main` branch instead.

And even if it was defined, it's still not published on npm: https://www.npmjs.com/package/@react-native-community/blur/v/3.6.1